cmd/storj-sim/main.go

```diff
@@ -59,6 +62,9 @@ func main() {
 
 	rootCmd.PersistentFlags().StringVarP(&flags.Postgres, "postgres", "", os.Getenv("STORJ_SIM_POSTGRES"), "connection string for postgres (defaults to STORJ_SIM_POSTGRES)")
 
+	// todo make redis required
+	rootCmd.PersistentFlags().StringVarP(&flags.Redis, "redis", "", "", "connection string for redis. If provided, storj-sim will use redis for all databases that support it.")
```

Comment:
Is "all databases that support it" meant to include the metainfoDB (pointerDB)? I think that would be supported by our redis client, but I don't think this PR makes that happen (also, I'm not sure that I would want that to happen).

Reply:
hmm yeah I agree that's not a great description. will update